darts/tests/models/forecasting/test_global_forecasting_models.py

```diff
@@ -126,31 +129,52 @@
                 "n_epochs": 10,
                 "pl_trainer_kwargs": tfm_kwargs["pl_trainer_kwargs"],
             },
-            100.0,
+            70.0,
         ),
         (
             NLinearModel,
             {
                 "n_epochs": 10,
                 "pl_trainer_kwargs": tfm_kwargs["pl_trainer_kwargs"],
             },
-            100,
+            50.0,
         ),
         (
             DLinearModel,
             {
                 "n_epochs": 10,
                 "pl_trainer_kwargs": tfm_kwargs["pl_trainer_kwargs"],
             },
-            100,
+            55.0,
```

Comment:
Did it change because of the reduction of the number of component and the number of epochs?